apps/crn-server/src/validation/research-output.validation.ts

```diff
@@ -169,7 +170,7 @@ const researchOutputPostRequestValidationSchema: JSONSchemaType<ResearchOutputPo
     },
     required: [
       'documentType',
-      'description',
+      'descriptionMD',
```

Comment:
I see you're making the markdown description compulsory instead of the old one - but the types still say both are required:
https://github.com/yldio/asap-hub/blob/313d4a49d59f59919a7f17a8d9244e591d3d5a3a/packages/model/src/research-output.ts#L292-L293

To be fair `JSONSchemaType` should enforce the correct behaviour but unfortunately it only does the opposite check (so if you mark a property as optional then you have to give a `nullable` parameter).